File: Semester 2/Object Oriented Programming/Lab 1/Ex1.java
import java.util.Scanner;
import java.util.Random;

public class Ex1
{
    public static void main(String args[])
    {
        Scanner read = new Scanner(System.in);
        
        System.out.println("Introduceti 2 numere intregi: ");
            int nr1 = read.nextInt();
            int nr2 = read.nextInt();

        System.out.println("Suma numerelor este: "+ (nr1+nr2));
        System.out.println("Produsul numerelor este: "+ (nr1*nr2));
        System.out.println("Catul impartirii "+ nr1 + " la " + nr2 + "este: "+ (nr1/nr2));
        System.out.println("Restul impartirii "+ nr1 + " la " + nr2 + "este: "+ (nr1%nr2));
 
        Random rnd = new Random();
        
        int random1 = rnd.nextInt(255);
        int random2 = rnd.nextInt(255);
        System.out.println("Valoarea "+nr1+" dupa inmultirea cu "+random1+" este: "+(nr1*random1));
        System.out.println("Valoarea "+nr2+" dupa inmultirea cu "+random2+" este: "+(nr2*random2));
    }
}

File: Semester 2/Advanced Programming Methods/Lab 7/Ex1.c
#include <stdio.h>

int gcd(int x, int y)
{
    if(!y)  return x;

    return gcd(y, x%y);
}


int main(void)
{
    int x, y;

    printf("Introduceti 2 numere:\n");
    printf("(int,int): ");
    scanf("%d%d", &x, &y);

    printf("gcd -> %d\n", gcd(x, y));

    return 0;
}